plugins/inputs/amqp_consumer/README.md

```diff
@@ -0,0 +1,35 @@
+# AMQP Consumer Input Plugin
+
+This plugin reads data from an AMQP Queue ([RabbitMQ](https://www.rabbitmq.com/) being an example) formatted in one of
```

Comment:
Here I think we should mention that this plugin is for the AMQP 0.9.1 protocol, and link to the RabbitMQ doc about it: https://www.rabbitmq.com/amqp-0-9-1-reference.html & https://www.rabbitmq.com/tutorials/amqp-concepts.html